bindings/ios/MEGASdk.h

```diff
@@ -5400,6 +5401,32 @@ typedef NS_ENUM(NSInteger, AffiliateType) {
  */
 - (void)killSession:(uint64_t)sessionHandle;
 
+/**
+ * @brief Returns the deadline to remedy the storage overquota situation
+ *
+ * This value is valid only when MegaApi::getUserData has been called after
```

Comment:
Better use the binding methods / enums
In example: [MegaSdk getUserData], 
MegaListener/MegaGlobalListener::onEvent ---> [MEGADelegate/MEGAGlobalDelegate onEvent:event:]

STORAGE_STATE_PAYWALL ---> StorageStatePaywall
MegaEvent::EVENT_STORAGE ---> EventStorage

Etc

Reply:
Got it